### ISSUE TITLE
Create expected output (MLP)

### ISSUE THREAD

**Issue by dwettstein** [dwettstein/pattern-recognition-2016#6]
- Plot showing the error-rate on the training and the validation set with respect to the
  training epochs.
- Test accuracy with the best parameters found during cross-validation.

http://stanford.edu/~mwaskom/software/seaborn/index.html

**Comment by dwettstein** [comment]
Done with 3b7992958712243e403d109d405dc1b889a9b413
